Dockerfile

```diff
@@ -18,6 +18,8 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
     && echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" | tee /etc/apt/sources.list.d/google-chrome.list \
 # Need backports for openjdk 11 package
     && echo 'deb http://deb.debian.org/debian stretch-backports main' > /etc/apt/sources.list.d/backports.list \
```

Comment:
openjdk-11-jdk is available in stable. I don't think we use stretch-backports for anything, and we should be able to remove it

Reply:
Remove it and see if the build still passes :)